src/api/VotingIrregularities.Domain.Seed/appsettings.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
-    "ConnectionStrings": {
-        "DefaultConnection": "TBR"
-    },
+  "ConnectionStrings": {
```

Comment:
Don't commit changes to `appsettings.json`. You can use `appsettings.development.json` for you local environment.

Reply:
Same as the upper comment